[xbmc][WindowsStore] Set store minimum version

Paxxi opened the pull request:
<!--- Provide a general summary of your change in the Title above -->

This Specifies the minimum runtime version of Win10 that we support as koying figured out.
For now this won't do anything at all but once we bump the sdk to the latest version it's important to keep this at least one version behind unless we really need something from the latest version
## Description
<!--- Describe your change in detail -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your change -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc -->

## Screenshots (if appropriate):

## Types of change
<!--- What type of change does your code introduce? Put an `x` in all the boxes that apply like this: [X] -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply like this: [X] -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

afedchin reviewed (approved):
If we want to support windows mobile on most devices, we will stay on 14393 as min for long time. because only 13 models received creators update.

Rechi commented:
Is the following unrelated to the version you set in this PR or should they stay the same?
https://github.com/xbmc/xbmc/blob/ce76a6d26cfbe6491f5542441a130ebb8aff1809/tools/windows/packaging/uwp/package.appxmanifest.in#L19

Also keep in mind this PR currently contains an unrelated commit (vswhere).

MartijnKaijser commented:
If we need to start defining the version at several places it make make sense to put it in version.txt or something

Paxxi commented:
Didn't know about that one. According to https://docs.microsoft.com/en-us/uwp/schemas/appxpackage/uapmanifestschema/element-targetdevicefamily it looks like they should match.
I'll see about having it in a version file so we have it in one place. 

Might want to do the same for the max version so that it matches the sdk used in buildsetup.bat. Not sure if that would cause any issues whenever a new windows is released though.

afedchin commented:
@Paxxi I think we should define min SDK ver in ArchSetup.cmake, at least for do not changing CMakeList.txt for every version bump.

@MartijnKaijser  no need to define this in version.txt because it's not a target SDK version but minimum supported, this may no change for couple years.

MartijnKaijser commented on tools/windows/packaging/uwp/package.appxmanifest.in:
```diff
@@ -16,7 +16,7 @@
 
   <Dependencies>
-    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.0.0" MaxVersionTested="10.0.0.0" />
+    <TargetDeviceFamily Name="Windows.Universal" MinVersion="@VS_MINIMUM_SDK_VERSION@" MaxVersionTested="10.0.0.0" />
```
Guess we should also bump MaxVersionTested to at least MinVersion? :)

afedchin commented on tools/windows/packaging/uwp/package.appxmanifest.in:
```diff
@@ -16,7 +16,7 @@
 
   <Dependencies>
-    <TargetDeviceFamily Name="Windows.Universal" MinVersion="@VS_MINIMUM_SDK_VERSION@" MaxVersionTested="10.0.0.0" />
+    <TargetDeviceFamily Name="Windows.Universal" MinVersion="@VS_MINIMUM_SDK_VERSION@" MaxVersionTested="@VS_MAXIMUM_SDK_VERSION@" />
```
please remove this change. VS will change this as needed

afedchin replied:
FYI you can download appx from the mirrot and check `AppxManifest.xml` inside

MartijnKaijser commented:
jenkins build this please

MartijnKaijser commented:
jenkins build this please

Rechi commented:
has to be squashed

Rechi reviewed (commented):
I can't say anything about this.

Paxxi committed:
Set store minimum version

This Specifies the minimum runtime version of Win10 that we support as koying figured out.

For now this won't do anything at all but once we bump the sdk to the latest version it's important to keep this at least one version behind unless we really need something from the latest version
